fake_switches/ssh_service.py

```diff
@@ -101,8 +102,11 @@ def getRSAKeys():
 """
     return host_public_key, host_private_key
 
+
 class SwitchSshService(object):
```

Comment:
Could this Class inherit from the new one so we don't have the code as duplicate?

Reply:
That stuff is deprecated.
IMO, next move would be to send it to the garbage.